ros2_control.rolling.repos

```diff
@@ -2,8 +2,8 @@ repositories:
   ros-controls/realtime_tools:
     type: git
     url: https://github.com/ros-controls/realtime_tools.git
-    version: foxy-devel
+    version: master
   ros-controls/control_msgs:
     type: git
     url: https://github.com/ros-controls/control_msgs.git
-    version: galactic-devel
+    version: humble
```

Comment:
Is there a reason why we don't have a consistent branch naming pattern for rolling branches? Otherwise we'll keep running into this.

Reply:
yerright

Reply:
```suggestion
    version: master
```

Reply:
I've created the branch, adjusted CI, etc